src/assets/locales/en/translation.json

```diff
@@ -436,7 +437,7 @@
     "join_discord": "Join our Discord",
     "ask_away": "and ask away!"
   },
-  "test_net": "TestNet",
+  "test_net": "Testnet",
```

Comment:
Actually, it's out of scope. I updated the text according to the industry convention. As it's minor thing. I did not create a separate PR.